.gitmodules

```
@@ -141,6 +141,9 @@
 [submodule "FEM_FrontISTR"]
 	path = FEM_FrontISTR
 	url = https://github.com/FrontISTR/FEM_FrontISTR.git
+[submodule "FEMbyGEN"]
+	path = FEMbyGEN
+	url = https://github.com/Serince/FEMbyGEN.git
```

This needs to specify the branch (since you are not using "master"):
```suggestion
	url = https://github.com/Serince/FEMbyGEN.git
	branch = main
```

---

I've renamed the branch as "master" to be compatible with other addons.

---

That is probably the harder way to go about it :) - the Addon Manager supports any branch name you like, there is no question of "compatibility." Now you have to make sure to update package.xml for the new branch.

---

Yes, It was already Master in package.xml file :). Because I modified it from another workbench